test/make-swappable-proposal.test.ts

```diff
@@ -62,20 +106,49 @@ describe("make swappable proposal", function () {
     }
   });
 
-  it("transfers COW to vCOW", async function () {
-    // Require that the mock in the test "executes successfully" has been
-    // called. This is done by observing that without the mock the transaction
-    // reverts.
+  // We want to test that each mock is used at its step to make sure that the
```

Comment:
This works, yes! 
Why did you decide against using the  function relayTokens and its event from
`OmniBridgeTransferSimulator`. 

I thought it would be a more "direct" way to test it